### ISSUE TITLE
ev3dev-buster

### ISSUE THREAD

**Issue by dlech** [ev3dev/ev3dev#1074]
Collecting some notes here:

Debian buster will be released sometime in 2019: https://lists.debian.org/debian-devel-announce/2018/04/msg00006.html

The armel architecture (used on EV3) is likely to be dropped from official Debian support: https://release.debian.org/buster/arch_qualify.html

**Comment by jabrena** [comment]
Yes, you have reason with the forecast, Debian releases every 2 years:
https://wiki.debian.org/DebianReleases?action=show&redirect=DebianRelease#Production_Releases

But `Debian Stretch`  is `LTS` and the support will finish in 2022 and Lego should release a new product in 2020 so we could have time to move to BullEye with the new product.
https://wiki.debian.org/DebianBullseye

**Lego Mindstorms Timeline:** 

1. January 1998: Lego Mindstorms RCX
2. August 2006: Lego Mindstorms NXT (After 8 years)
3. January 2013: Lego Mindstorms EV3 (After 7 years)
4. January 2020: ?? (After 7 years)

Source: https://www.lego.com/en-us/mindstorms/history

Juan Antonio

**Comment by dlech** [comment]
First ev3dev-buster snapshots are out: https://oss.jfrog.org/list/oss-snapshot-local/org/ev3dev/brickstrap/2019-03-04/

**Comment by jabrena** [comment]
Any change in the EV3Dev interface?

**Comment by dlech** [comment]
not yet...

**Comment by jabrena** [comment]
Maybe we should not change the interface until Stretch is official. It requires many changes and it could be better to have some documentation for the migration or at least, the list of issues affecting the interface.

**Comment by dlech** [comment]
There are no plans to make any changes to stretch.

**Comment by dlech** [comment]
Latest news says armel (and armhf) will be in buster.

https://lists.debian.org/debian-devel-announce/2019/04/msg00003.html

**Comment by PythonAtSea** [comment]
Lego has announced the 4th gen Lego Mindstrorms [here](https://www.lego.com/en-us/themes/mindstorms/about)
